test/DaedalusCompiler.Tests/SemanticErrorsTests.cs

```diff
@@ -55,13 +59,30 @@ private void AssertCompilationOutputMatch()
         {
             ParseData();
             var logger = new StringBufforErrorLogger();
+            logger.Log("");
```

Comment:
Do we need that?

Reply:
It's either this or adding `if` before `Assert.Equal(_expectedCompilationOutput, logger.GetBuffor().Trim());` because if we don''t log anything into the logger, logger.GetBuffor() will return null and null doesn't have Trim method.